src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -31,13 +30,9 @@ internal abstract class ConfigPropertyAccessor
         /// </summary>
         static ConfigPropertyAccessor()
         {
-#if CORECLR
             Instance = Platform.IsInbox
```

Comment:
"Instance = " is fixed.

For `Platform.IsInbox()` I'll open new PR (We have still 4 reference on it).